kuflow-engine-client-common/src/main/java/com/kuflow/engine/client/common/resource/UserActionWorkflowRequestResource.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021-present KuFlow S.L.
+ *
+ * All rights reserved.
+ */
+
+package com.kuflow.engine.client.common.resource;
+
+import java.util.UUID;
+
+public class UserActionWorkflowRequestResource extends AbstractResource {
```

Comment:
Where do you use this class?

Reply:
Is the input object of the action workflow method. I can reuse the WorkflowRequestResource and add the ActionvalueID, or create this new one. The object is a warpper of the ProcessId + UserActionId.

Reply:
oook, perfect, I get it.